xdsl/dialects/hw.py

```diff
@@ -656,20 +662,18 @@ class ModuleArg(NamedTuple):
 
     def get_module_type(self) -> ModuleType:
         return ModuleType(
-            [
-                ArrayAttr(
-                    tuple(
-                        ModulePort(
-                            (
-                                StringAttr(arg.port_name),
-                                arg.port_type,
-                                DirectionAttr(arg.port_dir),
-                            )
+            ArrayAttr(
+                tuple(
+                    ModulePort(
+                        (
+                            StringAttr(arg.port_name),
+                            arg.port_type,
+                            DirectionAttr(arg.port_dir),
```

Comment:
The title of the PR seems wrong, the init for `ModulePort` doesn't seem to be updated.

Reply:
Sure